addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -226,40 +246,15 @@ FormRenderer.include({
     },
     /**
      * @private
-     * @param {OdooEvent} ev
-     * @param {Object} ev.data
-     * @param {Attachment[]} ev.data.attachments
-     * @param {Thread} ev.data.thread
      */
-    _onChatterRendered(ev) {
-        if (!this.hasAttachmentViewer()) {
-            if (this.attachmentViewer) {
-                this.attachmentViewer.destroy();
-                this.attachmentViewer = undefined;
-                this._interchangeChatter();
-            }
-            return;
-        }
-        var self = this;
-        const thread = this.messaging.models['Thread'].insert({
-            id: this.state.res_id,
-            model: this.state.model,
-        });
-        if (this.attachmentViewer) {
-            // FIXME should be improved : what if somehow an attachment is replaced in a thread ?
-            if (
-                this.attachmentViewer.thread !== thread ||
-                this.attachmentViewer.attachments.length !== thread.attachmentsInWebClientView.length
-            ) {
-                this.attachmentViewer.updateContents(thread);
+    async _updateWebClientViewAttachmentViewContainer() {
+        const props = this._makeWebClientViewAttachmentViewContainerProps();
+        try {
+            await this.webClientViewAttachmentViewContainer.update(props);
+        } catch (error) {
+            if (error.message !== "Mounting operation cancelled") {
```

Comment:
looks like this error message doesn't exist anymore?

Reply:
Good catch, I guess I'll remove both of them.